TidepoolKit/TAPI.swift

```diff
@@ -432,6 +432,105 @@ public class TAPI {
         let valid: Bool
     }
 
+    // MARK: - Verify App
```

Comment:
nit: This looks pretty self contained (no new vars, just functions and types), and this file is pretty big. Maybe it'd be nice as an extension?